### PR TITLE
fix: sort pending multisig transaction correctly

### LINKED ISSUE #2450: Sorting pending tx in multisig is on bottom, should be on top
https://aeternity-blockchain.atlassian.net/browse/SW-1373

Describe the bug

Sorting pending tx in multisig is on bottom, should be on top. Reproducible on the release branch. 

Expected result

Steps to reproduce
1. Initiate multisig tx
2. Check the Tx list 

Image
Image

Environment selector

MacOS
iOS
Web wallet

### PR DESCRIPTION
**Pull request by CedrikNikita**
closes #2450

### CODE REVIEW

File: src/composables/multisigTransactions.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function useMultisigTransactions({ store }: IDefaultComposableOptions) {
         hash: txHash,
         tx: rawTx ? unpackTx(rawTx.tx) as any : undefined,
         isMultisigTransaction: true,
-        microTime: rawTx ? new Date(rawTx.createdAt).getTime() : undefined,
```
Comment (CedrikNikita) [inline src/composables/multisigTransactions.ts:91]: The `createdAt` property actually contains the time when the vault was created, not the proposal.

**Comment by Liubov-crypto** [comment]
LGTM

File: src/composables/multisigTransactions.ts (hunk 1 above)
Comment (martinkaintas) [reply src/composables/multisigTransactions.ts:91]: So we are setting the `microTime` to be `Date.now` in order to have the pending multisig transactions on top?

Comment (CedrikNikita) [reply src/composables/multisigTransactions.ts:91]: Yes, this is what we exactly doing on adding a regular pending transaction.

**Review by peronczyk — APPROVED** [review]

**Review by martinkaintas — APPROVED** [review]

**Pull request merged**